### PR TITLE
Update Exchange-Teams-interact.md

### PR DESCRIPTION
**Pull request by michelderooij**
Make bit more explicit (example) what doesn't work with Teams with Modern Hybrid, as it's limited to Free/Busy and MRS (irrelevant for this doc). I had 2 customers this week expecting calendar integration from Teams with their on-premises mailboxes works with Modern Hybrid. It doesn't, and they asked why the calendar button wasn't showing.

### CODE REVIEW

**Comment by ndambrosio** [comment]
Adding to @michelderooij, this is clearly documented in the [Microsoft Hybrid Agent article](https://docs.microsoft.com/en-us/exchange/hybrid-deployment/hybrid-agent#constraints), which references this article.  Michel's proposed changes would make this crisp and clear.

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/Exchange-Teams-interact.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Users hosted on Exchange Online or Exchange Dedicated vNext can use all the feat
 Users hosted on Exchange Online Dedicated (Legacy) must be synchronized to Azure Active Directory on Microsoft 365 or Office 365. They can create and join teams and channels, add and configure tabs and bots, and make use of the chat and calling features. However, they cannot modify profile pictures, manage meetings, access outlook contacts, or manage connectors.
 
 > [!IMPORTANT]
-> For integration with on-premises, it's highly recommended that you have an Exchange full Classic Hybrid deployment with Exchange Server 2016 or later to meet the requirements below. For more information about setting up a hybrid deployment, see [Exchange Server hybrid deployments](https://docs.microsoft.com/exchange/exchange-hybrid).
+> For integration with on-premises, it's highly recommended that you have an Exchange full Classic Hybrid deployment with Exchange Server 2016 or later. Modern Hybrid support is limited to Free/Busy and will not provide calender integration from Teams to mailboxes on-premises for example. For more information about setting up a hybrid deployment, see [Exchange Server hybrid deployments](https://docs.microsoft.com/exchange/exchange-hybrid).
```
Comment (JohanFreelancer9) [inline Teams/Exchange-Teams-interact.md:35]:
```suggestion
> For integration with on-premises, it's highly recommended that you have an Exchange full Classic Hybrid deployment with Exchange Server 2016 or later. Modern Hybrid support is limited to Free/Busy and will not provide calendar integration from Teams to mailboxes on-premises, for example. For more information about setting up a hybrid deployment, see [Exchange Server hybrid deployments](https://docs.microsoft.com/exchange/exchange-hybrid).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**